org.eclipse.jdt.debug.tests/tests/org/eclipse/jdt/debug/tests/core/ClasspathVariableTests.java

```diff
@@ -65,7 +65,6 @@ public void testJRELibResolution() throws CoreException {
 		assertNotNull("no default JRE", vm);
 		LibraryLocation[] libs = JavaRuntime.getLibraryLocations(vm);
 		assertTrue("no default libs", libs.length > 0);
-		assertEquals("Should resolve to location of local JRE", libs[0].getSystemLibraryPath().toOSString().toLowerCase(), resolved[0].getPath().toOSString().toLowerCase());
```

Comment:
What "resolved variables" do you have in mind. The issue here is that this test is actually wrong and does not test what the code actually implements. There is no reason the default VM is used here; it just happened to work on CI for a while because only 1 JRE was known to JDT.

Reply:
`resolved` variable above should be in the list of `libs`

Reply:
Unlike what the test has assumed,, the "libs" does *not* relate to the default VM; but to the VM that JDT think is best for the project. So the libs doesn't contain the (incorrectly) previously expected value. I don't think such an assert should and can be made to work in general.

Reply:
I can confirm that running the test locally that the project is only requiring 1.4 and it is satisfied by my local 1.8 but the default VM is Java 17 so the test fails if I try and find a match between libs and resolved.  Marking this removal as ok.